### PR TITLE
Quality of life updates

### PR DESCRIPTION
**Pull request by abnegate**
- Add PHPCS formatter + composer scripts
- Update to PHP 8
- Remove redundant minify params
- Covered by existing tests

### CODE REVIEW

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Well, this doesn't change anything with the functionality right? these are only housekeeping updates?

Also, let's update travis to run lint.

**Comment by abnegate** [comment]
@lohanidamodar Yep no functional changes. Added lint to travis as well 👍

**Review by lohanidamodar — APPROVED** [review]
Just to be clear these are only formatting and QOL changes. Looks good.

File: src/SDK/Language/Flutter.php [hunk 1]
```diff
@@ -44,29 +44,29 @@ public function getFiles()
...
             [
-				'scope'         => 'default',
-				'destination'   => '/lib/models.dart',
-				'template'      => 'dart/lib/models.dart.twig',
-				'minify'        => false,
-			],
+                'scope'         => 'default',
+                'destination'   => '/lib/models.dart',
+                'template'      => 'dart/lib/models.dart.twig',
+                'minify'        => false,
```
Comment (christyjacob4) [inline src/SDK/Language/Flutter.php:68]: Let's remove the `minify` key from here too

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**